ReviewReportDAO.php

```diff
@@ -121,7 +121,7 @@ public function getReviewReport($contextId)
             [(int) $contextId]
         );
         $interests = [];
-        while ($row = $assignedReviewerIds->next()) {
+        foreach ($assignedReviewerIds as $row) {
```

Comment:
The return from the query is a generator, which has a `next()` method, but it just advances and returns null 😁